Eevee.Sleep.Bot/Controllers/Mongo/MongoConst.cs

```diff
@@ -20,4 +20,12 @@ public static class MongoConst {
 
     public static readonly IMongoCollection<ActivationPresetModel> AuthActivationPresetCollection =
         AuthDatabase.GetCollection<ActivationPresetModel>("activationPreset");
+
+    private static readonly IMongoDatabase DiscordDatabase = Client.GetDatabase("role");
```

Comment:
Not this 😆 You misunderstood that.

What I mean was something closer to this (see `gpp/*`)
![image](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/assets/18443743/18be0188-7c3d-4762-9c23-2db53504bd0c)

Reply:
I was completely mistaken.
Fixed~
![image](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/assets/6148777/e85ed2ac-5db6-4408-858f-63123b39b435)

[c7bb2c2](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/c7bb2c2afd7c6b5ed54141b9b162ba890ff48870)